object-store-client-play-26/src/test/scala/uk/gov/hmrc/objectstore/client/play/PlayObjectStoreClientSpec.scala

```diff
@@ -61,24 +61,48 @@ class PlayObjectStoreClientSpec
 
   "putObject" must {
 
-    "store an object as Source" in {
-      val body      = s"hello world! ${UUID.randomUUID().toString}"
-      val location  = generateLocation()
-      val md5Base64 = Md5Hash.fromBytes(body.getBytes)
+    "store an object as Source with NotUsed materializer" in {
+      val body                                = s"hello world! ${UUID.randomUUID().toString}"
+      val location                            = generateLocation()
+      val md5Base64                           = Md5Hash.fromBytes(body.getBytes)
+      val source: Source[ByteString, NotUsed] = toSource(body)
 
       initPutObjectStub(location, statusCode = 201, body.getBytes, md5Base64)
 
-      osClient.putObject(location, toSource(body)).futureValue shouldBe (())
+      osClient.putObject(location, source).futureValue shouldBe (())
     }
 
-    "store an object as Source with known md5hash and length" in {
-      val body      = s"hello world! ${UUID.randomUUID().toString}"
-      val location  = generateLocation()
-      val md5Base64 = Md5Hash.fromBytes(body.getBytes)
+    "store an object as Source with Any materializer" in {
```

Comment:
`materializer` should be `materialised type`?

Reply:
Play calls the type Materializer and references are Materialiser everywhere. I think it makes sense to carry on with the Americanism here to avoid confusion.

Reply:
sorry didn't mean the spelling - one is the `Mat` materialized type, and the other is the `materializer` which is required to process the materialized type. the test label just looked like it was referring to the implicit `ActorMaterializer`.